components/copy_button.tsx

```diff
@@ -34,7 +33,7 @@ const CopyButton: React.FC<Props> = (props: Props) => {
     const tooltip = (
         <Tooltip id='copyButton'>
             <FormattedMessage
-                id={props.idMessage}
+                id={isCopied ? 'copied.message' : 'copy.block.message'}
```

Comment:
This just about works, but you need to wrap `copied.message` and `copy.block.message` in the `t` function from `utils/i18n` so that the translation tools can find them and add them to `i18n/en.json` automatically

Reply:
ah got it, this doesn't work because it's only clear what the id is at runtime.